### PR TITLE
Use babel to transform JS

### PR DESCRIPTION
**Pull request by yuvipanda**
After this, we can generally use most modern JS
without having to look at caniuse.com and see if
the feature is supported.

Ref https://github.com/jupyterhub/binderhub/issues/1373

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +16,8 @@
   "dependencies": {
     "bootstrap": "^3.4.0",
     "clipboard": "^2.0.8",
-    "jquery": "^3.6.0",
     "event-source-polyfill": "^1.0.25",
```
Comment (consideRatio) [inline package.json:19]: Is event source polyfill something we won't need using babel? I recall babel to me a lot about transforming code to be compatible.

File: babel.config.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```
Comment (consideRatio) [inline babel.config.json:3]:
```suggestion
}

```

**Review by consideRatio — COMMENTED** [review]
Sounds good to me!

File: package.json (hunk 1 above)
Comment (yuvipanda) [reply package.json:19]: We will definitely still need it, as it's not a *language* feature but a browser feature.

**Comment by consideRatio** [comment]
Wieee!!!

**Pull request merged**